web/rector.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
```

Comment:
Looks like this file can be run remotely by anonymous user. If we don't need this file to be deployed, can we remove it in CI script?

Reply:
Yes, it is publicly executable, but do you see any issues with that? https://pe-514-employees.pantheonsite.io/rector.php just returns a blank screen and looking at the code I don't see an obvious security issue with the file. The file is only temporary; once we've completed the D10 upgrade we can remove it.